### PR TITLE
Optionalize config and design, version-bump, minor changes

### PR DESCRIPTION
**Pull request by mokimo**
- Version bump on dependencies
- Changed spelling of an error message
- Updated readme.md

**- added default options for the design & image-services** 
Is that fine? Feedback appreciated!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ const publication = await liClient.getPublication({documentId: 1})
 
 We assume that you used the standard signup flow. This would give you a document with id 1. Of course you can change this id to any document in your project.
 
-4. Configure an image service
+4. OPTIONAL: Configure an image service (If you define nothing, the default settings will be used)
```
Comment (gabrielhase) [inline README.md]: are you sure? I don't think we have default settings...

File: README.md [hunk 2]
```diff
@@ -61,15 +61,18 @@ const config = {
 
 This configures Livindocs' image service (the same that is used on edit.livingdocs.io). You can of course also specify your own here or change the parameters for image rendering.
 
-5. Use a design
+5. OPTIONAL: Use a design (If none specified, our magazine example design will be used)
```
Comment (gabrielhase) [inline README.md]: same here: are you sure?

File: README.md [hunk 3]
```diff
@@ -61,15 +61,18 @@ const config = {
...
+
 ```
 const axios = require('axios')
-const designRes = await request.get('https://api.livingdocs.io/designs/living-times/0.0.11')
+const designRes = await request.get('https://api.livingdocs.io/designs/living-times/0.0.12')
```
Comment (gabrielhase) [inline README.md]: actually we are at 0.0.14 now

File: client/index.js [hunk 4]
```diff
@@ -29,7 +29,7 @@ function validateConfig (config) {
   }
 
   if (!config.accessToken) {
-    throw new Error('API token missing')
+    throw new Error('acessToken missing')
```
Comment (gabrielhase) [inline client/index.js]: how about "API accessToken is missing"?

File: document/index.js [hunk 5]
```diff
@@ -1,7 +1,36 @@
 const utils = require('./utils')
+const fetch = require('node-fetch')
+
+const defaultConfig = {
```
Comment (gabrielhase) [inline document/index.js]: ah, I see :) thx!

Comment (mokimo) [reply document/index.js]: Do i still need to answer the questions above about the defaults? :D
Spares some of the boilerplate and we "suggest" to use those ones in the readme anyway!

File: README.md (hunk 3 above)
Comment (mokimo) [reply README.md]: 0.0.12 is the latest that we got online though
https://api.livingdocs.io/designs/living-times/0.0.14 doesn't exist

Comment (gabrielhase) [reply README.md]: ah, good you write this. this will be turned off very soon. our new design server is `https://server.livingdocs.io`

**Pull request merged**